doc/memos/rdm0.md

```diff
@@ -0,0 +1,146 @@
+- RIOT Maintainers Community (RIOT-MC)  
+- Developer Memo: 0  
```

Comment:
Mh, though I prefer also an enumerated *naming* of the file, I'm not sure about redundantly stating the number here again. Considering https://github.com/RIOT-OS/RIOT/pull/6191#issuecomment-270870741, this might lead to some collisions if someone messes up the maintenance/review.